packages/react/src/auto/AutoForm.ts

```diff
@@ -131,15 +131,15 @@ export const useAutoForm = <
     resolver: useValidationResolver(metadata),
     send: () => {
       const fieldsToSend = fields
-        .map(({ path }) => path)
-        .filter((item) => {
+        .filter(({ metadata }) => {
           if (props.include) {
-            return props.include?.includes(item);
+            return props.include?.includes(metadata.apiIdentifier);
           } else if (props.exclude) {
-            return !props.exclude?.includes(item);
+            return !props.exclude?.includes(metadata.apiIdentifier);
           }
           return true;
-        });
```

Comment:
`path` is a dot-separated string with the format `model.field` (e.g., `widget.name`), but the strings inside `include` and `exclude` are expected to be field API identifiers.

So, when the `include` prop exists, the `includes(item)` is always `false` and none of the fields are included in the payload.